packages/components/tests/dummy/app/templates/index.hbs

```diff
@@ -77,6 +77,41 @@
             Dropdown
           </LinkTo>
         </li>
+        <li class="dummy-paragraph">
```

Comment:
[issue] these should be sorted alphabetically.

Reply:
I'll make a commit for this

Reply:
Looking at this more I suspect the ordering here was actually intentional as its logically grouping the sub components together so that radio/toggle/checkbox are together. Made them alphabetical anyhow but something to think about for future documentation needs.